builtin/rebase.c

```diff
@@ -12,6 +12,11 @@
 #include "refs.h"
 #include "dir.h"
 #include "packfile.h"
+#include "lockfile.h"
```

Comment:
Whoops, I failed to say this: do we need to include the `lockfile.h` header? I *think* this is a left-over from copy-editing `do_reset()` in this here file.